kv/kvserver: TestStoreScanIntents failed

kv/kvserver.TestStoreScanIntents [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3390304&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3390304&tab=artifacts#/) on master @ [b0379ad4b164547a64eafac8b782f88794e85be1](https://github.com/cockroachdb/cockroach/commits/b0379ad4b164547a64eafac8b782f88794e85be1):


```
=== RUN   TestStoreScanIntents
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestStoreScanIntents813441600
    test_log_scope.go:74: use -show-logs to present logs inline
    store_test.go:2194: 0: scan failed to finish after 1s
    store_test.go:2209: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestStoreScanIntents813441600
--- FAIL: TestStoreScanIntents (1.17s)
```
<details><summary>Reproduce</summary>
<p>
To reproduce, try:

```bash
make stressrace TESTS=TestStoreScanIntents PKG=./pkg/kv/kvserver TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
/cc @cockroachdb/kv spencerkimball
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStoreScanIntents.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/kv/kvserver/store_test.go
+++ b/pkg/kv/kvserver/store_test.go
@@ -2098,7 +2098,7 @@ func TestStoreScanIntents(t *testing.T) {
 			errChan <- pErr
 		}()
 
-		wait := 1 * time.Second
+		wait := testutils.DefaultSucceedsSoonDuration
 		if !test.expFinish {
 			wait = 10 * time.Millisecond
 		}

